xarray/test/test_conventions.py

```diff
@@ -494,6 +494,12 @@ def test_dataset_repr_with_netcdf4_datetimes(self):
         ds = decode_cf(Dataset({'time': ('time', [0, 1], attrs)}))
         self.assertIn('(time) datetime64[ns]', repr(ds))
 
+        # this should not throw a warning (GH1111)
```

Comment:
Right, that's much easier! I was assuming that the tutorial files were also used for your tests. Salem's test suite is based on real files (similar to xarray's) and I've never had troubles with it, but I agree that it's not optimal